src/sagemaker/mxnet/estimator.py

```diff
@@ -78,8 +82,8 @@ def create_model(self, model_server_workers=None):
         return MXNetModel(self.model_data, self.role, self.entry_point, source_dir=self.source_dir,
                           enable_cloudwatch_metrics=self.enable_cloudwatch_metrics, name=self._current_job_name,
                           container_log_level=self.container_log_level, code_location=self.code_location,
-                          py_version=self.py_version, model_server_workers=model_server_workers,
-                          sagemaker_session=self.sagemaker_session)
+                          py_version=self.py_version, framework_version=self.framework_version,
```

Comment:
We probably want an extra assertion in the appropriate unit test that we pass along the framework version correctly to the created model (for both TF and MXNet).

Reply:
https://github.com/aws/sagemaker-python-sdk/pull/82/files#diff-8e5bb247678c2ede89b070ed8c1926f2R192

We already do this indirectly.

I guess it would be good to have a unit test that tests just model creation.